MANIFEST.in

```diff
@@ -1,6 +1,6 @@
 recursive-include merlin/data *.yaml *.py
 recursive-include merlin/server *.yaml *.py
-recursive-include merlin/examples *.py *.yaml *.c *.json *.sbatch *.bsub *.txt *.temp
+recursive-include merlin/examples *.py *.yaml *.c *.json *.sbatch *.bsub *.txt *.temp *.sh
```

Comment:
Maybe this should just be * (untested)

Reply:
I just did a quick glance through of the example workflow folders and I think you're right. I'm going to give it a shot before this gets merged.

Reply:
This change worked. Just pushed it.